plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -135,6 +135,23 @@ public LocationLayerPlugin(@NonNull MapView mapView, @NonNull MapboxMap mapboxMa
     initialize();
   }
 
+  /**
+   * Construct a LocationLayerPlugin
+   *
```

Comment:
Would you mind adding a note here that this constructor will use the built-in `LocationEngine`?

Reply:
Added the note to the two new constructors added in this pr.